mixer/pkg/perf/controller.go

```diff
@@ -15,13 +15,20 @@
 package perf
 
 import (
+	"fmt"
 	"net"
 	"net/http"
 	"net/rpc"
+	"time"
+
+	errorsutil "github.com/onsi/gomega/gstruct/errors"
 
 	"istio.io/istio/pkg/log"
 )
 
+// TODO(lichuqiang): modify defaultTimeout accordingly.
+const defaultTimeout = time.Duration(30 * time.Second)
```

Comment:
This is likely to be too restrictive. When running perf tests, I've used 1 or 5 minute intervals to get some stable results.